packages/java-packages/snippet-replacer-maven-plugin/pom.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.microsoft.azure.tools</groupId>
```

Comment:
Should this be under, e.g. `com.azure.tools` to match our track two naming?

Reply:
I'm ok with that. Got `com.microsoft.azure.tools` from Wes and Jim on discussion, then Mitch also suggested `com.azure.tools`. I guess I know which way i'm going!

Reply:
Updated to `com.azure.tools`